### ISSUE TITLE
ruff and black disagree on `.pyi` format for one line `def`

### ISSUE THREAD

**Issue by messense** [charliermarsh/ruff#2763]
```python
class DummyClass:
    @staticmethod
    def get_42() -> int: ...
```

ruff gives `test-crates/pyo3-pure/pyo3_pure.pyi:3:5: E704 Multiple statements on one line (def)` while black formats it back to one line if you change it to

```python
class DummyClass:
    @staticmethod
    def get_42() -> int:
        ...
```

**Comment by charliermarsh** [comment]
Need to figure out how to handle this because Flake8 and pycodestyle actually _do_ flag these as errors (they just don't enable that rule by default, AFAICT).

**Comment by charliermarsh** [comment]
I'm going to either disable this rule by default or remove it entirely.
